frontend/src/components/ChatMessage.js

```diff
@@ -1,5 +1,6 @@
 import bot from '../media/img/icon-bot.png'
 import user from '../media/img/icon-user.png'
+import ChatbotLoading from './ChatbotLoading';
```

Comment:
Is there another ChatbotLoading component file that needs to be committed?

Reply:
Sorry, I forgot to add it.